code/datums/mind.dm

```diff
@@ -68,7 +68,7 @@
 
 //SKYRAT CHANGES
 	var/appear_in_round_end_report = TRUE  //Skyrat change
-	var/accept_ERG = FALSE
+	var/accept_ERG = TRUE
```

Comment:
Azarak told me to leave this on false by default...

Reply:
Due to the fact we're sending those that opt out to the cafe, it makes sense just for the beginning so everyone isn't teleported to the ghost cafe.